CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 3.0.2)
+if(CMAKE_MAJOR_VERSION LESS 3)
+  cmake_minimum_required(VERSION 2.6)
+else()
+  cmake_minimum_required(VERSION 2.8.12)
```

Comment:
What is the reason to support a cmake version from 10 years ago? :eyes:

Reply:
It's still used on one of the systems on which I'm testing roc (some old raspberry pi). While there's no big maintaining burden, I prefer to keep compatibility with older systems, because often people are too lazy to update them, like me :)